tests/jvm/src/test/resources/example/Types.scala

```diff
@@ -111,6 +111,6 @@ object Test/*<=types.Test.*/ {
     final val bool/*<=types.Test.Literal.bool.*/ = true
     final val unit/*<=types.Test.Literal.unit.*/ = ()
     final val javaEnum/*<=types.Test.Literal.javaEnum.*/ = java.nio.file.LinkOption/*=>java.nio.file.LinkOption#*/.NOFOLLOW_LINKS/*=>java.nio.file.LinkOption#NOFOLLOW_LINKS.*/
-    final val clazzOf/*<=types.Test.Literal.clazzOf.*/ = classOf[Option[Int]]
+    final val clazzOf/*<=types.Test.Literal.clazzOf.*/ = classOf/*=>scala.Predef.classOf().*/[Option[Int]/*=>scala.Option#*/]
```

Comment:
I would expect the occurrence position to enclose the `Option` identifier 🤔 
```suggestion
    final val clazzOf/*<=types.Test.Literal.clazzOf.*/ = classOf/*=>scala.Predef.classOf().*/[Option/*=>scala.Option#*/[Int]]
```

Reply:
Another test case to handle: qualifiers
```scala
classOf[scala.Option[Int]]
```

Reply:
Another test case: renamed imports
```scala
import scala.concurrent.{Future => Fut}

classOf[Fut[Int]]
```

Reply:
I've added tests, but not sure if they are in the right place.